README.md

```diff
@@ -26,57 +26,26 @@ Differentiating features include:
 [YAML]: http://yaml.org
 [Prometheus]: http://prometheus.io
 
-## Architecture
+## Why Chihaya?
 
-### Diagram
+Chihaya is built for developers who need to deploy a BitTorrent tracker, but have additional requirements than general purpose public trackers like those operated by [OpenBitTorrent].
+These requirements could be things like authentication, auditability, or features that take advantage of assumptions about their deployment environments.
+Chihaya's middleware framework enables simple and flexible integration point that abstracts the BitTorrent tracker protocols.
+The most common use case for Chihaya is integration with the deployment of cloud software to private networks.
```

Comment:
That sounds weird to me. Would you describe quay as a platform for the deployment of cloud software to private networks?